docs/framework/abbreviation-plugin-tutorial/abbreviation-plugin-level-1.md

```diff
@@ -19,7 +19,7 @@ If you want to see the final product of this tutorial before you plunge in, chec
 
 The easiest way to set up your project is to grab the starter files from our [Github repository for this tutorial](https://github.com/ckeditor/ckeditor5-tutorials-examples/tree/main/abbreviation-plugin/starter-files). We gathered all the necessary dependencies there, including some CKEditor 5 packages and other files needed to build the editor.
 
-The editor has already been created in the `app.js` file with some basic plugins. All you need to do, is clone the repository, run the `npm install` command, and you can start coding right away.
+The editor has already been created in the `app.js` file with some basic plugins. All you need to do, is clone the repository, navigate into one of the practice directories (i.e. part-1), run the `npm install` command, and you can start coding right away. See the [README.md](https://github.com/ckeditor/ckeditor5-tutorials-examples/tree/main/README.md) file for more detailed setup information.
```

Comment:
```suggestion
The editor has already been created in the `app.js` file with some basic plugins. All you need to do is clone the repository, navigate to the [starter-files directory](https://github.com/ckeditor/ckeditor5-tutorials-examples/tree/main/abbreviation-plugin/starter-files), run the `npm install` command, and you can start coding right away.
```

Good catch! One step was missing. We are in the "Let's start" section, so at this point it would be better to link to the `starter-files` directory. The readme.md file doesn't seem to contain any additional information related to setup compared to what is in the guide I think.

Reply:
Thanks.  I'm new here... do I need implement the suggestions above, or have you already applied them to the merge request?